src/controllers/user.js

```diff
@@ -15,14 +16,15 @@ export const create = async (req, res) => {
     const existingUser = await User.findByEmail(userToCreate.email)
 
     if (existingUser) {
-      return sendDataResponse(res, 400, { error: 'Email already in use' })
+      return sendDataResponse(res, 400, { error: ERR.EMAIL_IN_USE })
     }
 
     const createdUser = await userToCreate.save()
 
     return sendDataResponse(res, 201, createdUser)
   } catch (error) {
-    return sendMessageResponse(res, 500, 'Unable to create new user')
+    console.error(ERR.UNABLE_TO_CREATE_USER)
```

Comment:
Actually, come to think of it, I believe what Lewis was asking for is to get both the user friendly error _and_ the actual error that was thrown logged to the console. Could you change this to also log out the actual error as well as the user-friendly error please?

The actual error will include the technical information a dev might need

Reply:
The imported error message is identical to the 'user friendly' error message -- and must remain generic because it's a catch all for registration errors. 

But senior devs know best, sir, so I've updated accordingly.